File: trunk/ShadowFrameworkLite20_OpenGL20/src/shadow/pipeline/openGL20/tutorials/bitmapsExample/PerlinNoise2.java
package shadow.pipeline.openGL20.tutorials.bitmapsExample;

import shadow.image.SFBitmap;
import shadow.image.bitmaps.SFSimplePerlinNoise;

public class PerlinNoise2 {

	public static SFBitmap generateBitmap() {
		float[] weights={0.6f,0.2f,0.1f,0.05f,0.05f,0.05f,0.05f};
		
		SFSimplePerlinNoise.randomizeFunction();
		SFBitmap bitmap=new SFSimplePerlinNoise(100, 100, weights, false);
		return bitmap;
	}

}

File: trunk/ShadowFrameworkJS/SF20OpenGL20Graphics/src/shadow/pipeline/openGL20/tutorials/bitmapsExample/PerlinNoise2.js

function PerlinNoise2(){
}

PerlinNoise2.prototype = {

	generateBitmap:function(){
		 float[] weights={
		0.6f,0.2f,0.1f,0.05f,0.05f,0.05f,0.05f;
	}
		SFSimplePerlinNoise.randomizeFunction();
		 SFBitmap  bitmap = new  SFSimplePerlinNoise(100, 100, weights, false);
		return ,bitmap;
	}

};